README.md

```diff
@@ -1,96 +1,125 @@
 # Swift Numerics
   
 ## Introduction
+
 Swift Numerics provides a set of modules that support numerical computing in Swift.
 These modules fall broadly into two categories:
 
 - API that is too specialized to go into the standard library, but which is sufficiently general to be centralized in a single common package.
 - API that is under active development toward possible future inclusion in the standard library.
 
-There is some overlap between these two categories, and API that begins in the first category may migrate to the second as it matures and new uses are discovered.
+There is some overlap between these two categories, and an API that begins in the first category may migrate into the second as it matures and new uses are discovered.
+
+Swift Numerics modules are fine-grained.
+For example, if you need support for Complex numbers, you can import ComplexModule¹ as a standalone module:
 
-Swift Numerics modules are fine-grained; if you need support for Complex numbers, you can import ComplexModule¹ without pulling in everything else in the library as well:
 ```swift
 import ComplexModule
 
 let z = Complex<Double>.i
 ```
-However, there is also a top-level `Numerics` module that simply re-exports the complete public interface of swift-numerics:
+
+There is also a top-level `Numerics` module that re-exports the complete public interface of Swift Numerics:
+
 ```swift
 import Numerics
 
-// All swift-numerics API is now available
+// The entire Swift Numerics API is now available
 ```
 
 Swift Numerics modules have minimal dependencies on other projects.
+
 The current modules assume only the availability of the Swift and C standard libraries and the runtime support provided by compiler-rt.
-Future expansion may assume the availability of other standard interfaces such as [BLAS (Basic Linear Algebra Subprograms)](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK (Linear Algebra Package)](https://en.wikipedia.org/wiki/LAPACK), but modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
+
+Future expansion may assume the availability of other standard interfaces, such as [BLAS (Basic Linear Algebra Subprograms)](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK (Linear Algebra Package)](https://en.wikipedia.org/wiki/LAPACK).
+But modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
```

Comment:
```suggestion
, but modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
```

Reply:
Thanks for reviewing @xwu, appreciate it. If you think it's important to have this section as one long sentence, I can definitely make this change!

I noticed that in GitHub's web UI, the whole paragraph takes up three lines. If we merge these sentences, they become a one long thing spanning three lines:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/19637339/92313384-2b292780-efc3-11ea-8848-93309819c8ab.png">

---

Here's the reasoning behind simplifying a large sentence into two shorter ones (these are guidelines, not rules):

From Google Technical Writing - [Short sentences](https://developers.google.com/tech-writing/one/short-sentences):

> "Shorter documentation reads faster than longer documentation."
> 
> "Finding the shortest documentation implementation takes time but is ultimately worthwhile. Short sentences communicate more powerfully than long sentences, and short sentences are usually easier to understand than long sentences."

From Microsoft Style Guide - [Writing Tips](https://docs.microsoft.com/en-us/style-guide/global-communications/writing-tips):

> "These practices will help localizers and customers.
> "Write short, simple sentences. Punctuating a sentence with more than a few commas and end punctuation usually indicates a complex sentence. Consider rewriting it or breaking it into multiple sentences."

https://developers.google.com/tech-writing/one/short-sentences

Reply:
This is one thought, not two, so it needs to be one sentence.

Reply:
Ok @xwu! Will make the changes